app/controllers/history_controller.rb

```diff
@@ -0,0 +1,49 @@
+class HistoryController < ApplicationController
```

Comment:
I didn't wanna call this PastLinksController since we've already got one of those (but it's user-scoped). I'm down to rename this as LinkHistoryController if you think it's not specific enough